graphql_client_codegen/src/deprecation.rs

```diff
@@ -11,13 +11,13 @@ pub enum DeprecationStatus {
 }
 
 #[derive(Debug, PartialEq)]
-pub(crate) enum DeprecationStrategy {
+pub enum DeprecationStrategy {
     Allow,
     Deny,
     Warn,
 }
 
-pub(crate) fn extract_deprecation_strategy(
+pub fn extract_deprecation_strategy(
```

Comment:
since this is specific to the derive attributes I would be in favour of moving this function back to graphql_query_derive (in a subsequent PR).